tests/Adapter.test.js

```diff
@@ -6,7 +6,7 @@ describe("Adapter", () => {
   let gAdapter;
 
   const tableName = "fake_item";
-  const dropQuery = `DROP TABLE IF EXISTS ${tableName}`;
+  const dropQuery = (tableName) => `DROP TABLE IF EXISTS ${tableName}`;
```

Comment:
Since it's not a query anymore, could you rename it to `getDropQueryForTable()`?